libmscore/glissando.cpp

```diff
@@ -286,10 +292,9 @@ void Glissando::layout()
          // but ignore graces after, as they are not the first note of the system,
          // even if their segment is the first segment of the system
          && !(cr2->noteType() == NoteType::GRACE8_AFTER
-            || cr2->noteType() == NoteType::GRACE16_AFTER || cr2->noteType() == NoteType::GRACE32_AFTER)
+         || cr2->noteType() == NoteType::GRACE16_AFTER || cr2->noteType() == NoteType::GRACE32_AFTER)
```

Comment:
IMHO that indent should stay here, to repflect the parens

Reply:
Hm... That's the result of passing uncrustify with the config I found in the discussion about proposed new coding style. So I cannot trust this config. I'll fix it.

Reply:
Something is wrong with the two commands you gave me for rebasing and pushing because every time I use it, it pushes the first commit and, thus, trashes the fix I just did.

Reply:
I don't know how to undo that. GitKraken doesn't show this additional commit and my local source shows the fix is there.

Reply:
I found this in Git push doc:
"-f
--force
Usually, the command refuses to update a remote ref that is not an ancestor of the local ref used to overwrite it. Also, when --force-with-lease option is used, the command refuses to update a remote ref whose current value does not match what is expected.

This flag disables these checks, and can cause the remote repository to lose commits; use it with care."

Reply:
Indeed after a rebase a `-push -f` is needed,